docs/bugs.json

```diff
@@ -1,4 +1,17 @@
 [
+    {
+        "uid": "SOL-2023-7",
+        "name": "MissingSideEffectsOnSelectorAccessBug",
+        "summary": "Accessing the .selector member on complex expressions leaves the expression unevaluated in legacy code generation.",
+        "description": "When accessing the .selector member on complex expressions, like the return of a function call, such expressions will not be evaluated in the legacy code generation, leading to a different behavior when compiling with and without via-ir.",
```

Comment:
The entry should use double backticks for code snippets (see the other entries for reference).

Also not sure whether to emphasize the "different behaviour" rather than calling the behaviour broken in legacy, but via-IR unaffected :-).